src/components/o-testimonials/js/testimonials.js

```diff
@@ -40,17 +41,22 @@ class Testimonials extends UiEvents {
     this.slider = this.wcNode.querySelector(this.options.slider);
     this.autoRotateDisabled = getAttribute(this.wcNode, this.options.autoRotateDisabled);
     this.autoRotateTimeInMiliseconds = getAttribute(this.wcNode, this.options.autoRotateTime);
+    this.showAllInline = getAttribute(this.wcNode, this.options.showAllInline);
     if (!this.autoRotateTimeInMiliseconds) {
       this.autoRotateTimeInMiliseconds = 5000;
     }
     this.calculateContainerMinHeight();
-    this.hideAllSlides();
-    if (this.slides.length < 2) {
+    // if there is only 1 slide or the option to show all slides is enabled hide the slide controls.
+    if (this.slides.length < 2 || this.showAllInline) {
```

Comment:
May it's nicer to invert the logic here and only call `showControls` if they are needed 🤔

Reply:
Updated.